product_pricelist_direct_print/wizards/product_pricelist_print.py

```diff
@@ -226,7 +226,8 @@ def get_last_ordered_products_to_print(self):
         if not partner and self.partner_count == 1:
             partner = self.partner_ids[0]
         orders = self.env["sale.order"].search(
-            self._get_sale_order_domain(partner), order="date_order desc"
+            self._get_sale_order_domain(partner.commercial_partner_id),
```

Comment:
Could it be interesting to make this configurable in the wizard?

Reply:
I think that the change makes sense and it's not necessary to add an extra configuration.